### PR TITLE
Fix {{raw}} behavior in template

### PR DESCRIPTION
**Pull request by wejendorp**
Fails to render anything, to verify the broken behavior.

### CODE REVIEW

File: templates/kapeta/block-type-frontend/ai-templates/ApplicationScreen.tsx.hbs [hunk 1]
```diff
@@ -23,7 +23,7 @@ export const ApplicationScreen = () => {
         ];
     }, []);
 
-    {{#raw}}
+    {{{{raw}}}}
```
Comment (andertun) [inline templates/kapeta/block-type-frontend/ai-templates/ApplicationScreen.tsx.hbs:26]: Is escaping `{` necessary now? If not can we replace all `&#123;\{` with `{{` ?

File: package.json [hunk 2]
```diff
@@ -32,7 +32,17 @@
...
+        "@kapeta/codegen-target",
+        "@kapeta/kaplang-core",
+        "@kapeta/nodejs-process",
+        "@kapeta/nodejs-utils",
+        "@kapeta/schemas",
+        "@kapeta/ui-web-types",
+        "@types/lodash",
+        "lodash",
+        "prettier"
+    ],
```
Comment (andertun) [inline package.json:45]: I think this is equivalent to `"bundleDependencies": true`

**Review by andertun — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (wejendorp) [reply package.json:45]: Oh yea I don't know why that made it in there tbh 🤔

File: templates/kapeta/block-type-frontend/ai-templates/ApplicationScreen.tsx.hbs (hunk 1 above)
Comment (wejendorp) [reply templates/kapeta/block-type-frontend/ai-templates/ApplicationScreen.tsx.hbs:26]: Yes, actually that's definitely needed. I forgot to push it.

**Pull request merged**